src/world.c

```diff
@@ -62,14 +62,16 @@ void create_world(int p, int q, world_func func, void *arg) {
                     }
                 }
             }
-            // stars
+	    ///
+	    /// Show stars int the sky as long as SHOW_STARS value remains = 1
+	    ///
             if (SHOW_STARS) {
-		// stars show from y 100-150
+		/// stars show from y 100-150
 		for (int y = 100; y < 150; y++) {
-		    // simplex3 parameters: : x, y, z, octaves, persistance, lacunarity
-		    // uses perlin noise function simplex3 from noise.c (deps/noise/noise.c)  
+		    /// simplex3 parameters: : x, y, z, octaves, persistance, lacunarity
```

Comment:
I think if you put a doxygen comment in the simplex3 function located in noise.c then when the documentation is generated it will be clear what the parameters mean so we don't have to have comments in this for loop. Optional if you have the time, I think it would look more neat- let me know what you think.

Reply:
I might do this, might not... I didn't write that function and tbh don't completely understand what is going on with it so I probably won't be adding comments to noise.c

Reply:
Totally understandable, up to you I think it speaks for itself either way